docs/resources/Channel.md

```diff
@@ -1185,10 +1185,10 @@ When called on a `GUILD_TEXT` channel, creates a `GUILD_PUBLIC_THREAD`. When cal
 
 ###### JSON Params
 
-| Field                   | Type    | Description                                                                                                         |
-|-------------------------|---------|---------------------------------------------------------------------------------------------------------------------|
-| name                    | string  | 1-100 character channel name                                                                                        |
-| auto_archive_duration\* | integer | duration in minutes to automatically archive the thread after recent activity, can be set to: 60, 1440, 4320, 10080 |
+| Field                   | Type     | Description                                                                                                         |
+|-------------------------|----------|---------------------------------------------------------------------------------------------------------------------|
+| name                    | string   | 1-100 character channel name                                                                                        |
+| auto_archive_duration\* | ?integer | duration in minutes to automatically archive the thread after recent activity, can be set to: 60, 1440, 4320, 10080 |
```

Comment:
Just to be clear, this means it accepts `null`. You might have meant to put the question mark on the name itself, which means that the key is optional.